functions/src/modules/SE.functions.ts

```diff
@@ -88,3 +88,58 @@ export const processNewAllotment = functions.database
     });
     return 1;
   });
+
+const serialRegex = "^[a-zA-Z]+-\\d+";
+
+const updateChunk = async (path: string, taskId: string) => {
+  return await db.ref(path).update({
+    taskId
+  });
+};
+
+const makeTaskId = (fileName: string, index: number) => {
+  return `${fileName.match(serialRegex)[0]}-${index}`;
+};
+
+export const createTaskFromChunks = functions.database.ref(
+    "/sound-editing/chunks/{listId}/{fileName}/{index}"
+).onCreate(async (snapshot, { params: { fileName, listId, index } }) => {
+  const {
+    continuationTo,
+    continuationFrom,
+    processingResolution = "",
+    taskId,
+    beginning,
+    ending
+  } = snapshot.val();
+  if (continuationTo || processingResolution.toLowerCase() !== "ok" || taskId) return;
+  const chunkDuration = ending - beginning;
+  const chunkTaskId = makeTaskId(fileName, index);
+  let duration;
+
+  if (continuationFrom) {
+    const response = await db
+        .ref(`/sound-editing/chunks/${listId}/${continuationFrom}`)
+        .orderByKey()
+        .limitToLast(1).once("value");
+    const chunks = response.val();
+    const lastChunk = chunks[chunks.length - 1];
+    if (lastChunk.continuationTo === fileName) {
```

Comment:
This is very important check. Better to negate it and issue a warning message, aborting further processing of the current file.